CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 #### Enhancements
 
+* Add `indented_cases` support to `switch_case_alignment` rule.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes